src/tools/index.py

```diff
@@ -26,19 +30,38 @@ def get_index(index: str) -> Dict:
         def create_index(index: str, body: Optional[Dict] = None) -> Dict:
             """
             Create a new index.
-            
+
             Args:
                 index: Name of the index
                 body: Optional index configuration including mappings and settings
             """
             return self.search_client.create_index(index=index, body=body)
 
+        @mcp.tool(description="Create new index with the same mapping as existing_index")
+        def create_index_copy(index: str, existing_index: str) -> list[TextContent]:
```

Comment:
Could you please update `create_index_copy` in the README?
https://github.com/cr7258/elasticsearch-mcp-server?tab=readme-ov-file#index-operations